src/main/java/com/acrolinx/proxy/AcrolinxProxyHttpServlet.java

```diff
@@ -38,9 +43,11 @@
  */
 public class AcrolinxProxyHttpServlet extends HttpServlet {
   public static final int BUFFER_SIZE = 8_192;
+  public static final String DEFAULT_CONNECT_TIMEOUT_MILLIS_STRING = "-1";
   // TODO: Set this path in context of your servlet's reverse proxy implementation
   public static final String PROXY_PATH = "acrolinx-proxy-sample/proxy";
   private static final String ACROLINX_BASE_URL_HEADER = "X-Acrolinx-Base-Url";
+  private static final Logger LOGGER = LoggerFactory.getLogger(AcrolinxProxyHttpServlet.class);
```

Comment:
We don't need a logger and a LOGGER, one should be enough

Reply:
You're right, I totally missed the presence of the logger variable while refactoring.